bin/riemann-dir-files

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+# Gets the number of files present on a directory and submits it to riemann
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+
+class Riemann::Tools::DirFiles
```

Comment:
I just named it DirFiles to keep a naming convention with the existent "DirSpace" class (riemann-dir-space), as it's a similar function over a single folder with a different purpose of course. We can rename it if you wish, it's your call. Just reply me here what you prefer and I will change the class name (also the filename).